wedlocks/config-builder.js

```diff
@@ -50,6 +50,6 @@ console.log(
     unlockEnv,
     wedlocksPrivateKey,
     sender:
-      process.env.SMTP_FROM_ADDRESS || 'Unlock Labs <hello@unlock-protocol.com>', // TODO: can we do better eventually?
+      process.env.SMTP_FROM_ADDRESS || 'hello@unlock-protocol.com',
```

Comment:
I think this is what was creating the errors you were seeing.

Reply:
Is `SMTP_FROM_ADDRESS` also changed from GitHub variables?